app/assets/scripts/components/documents/single-view/common/variables-table.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { EmptySection } from '../document-body';
+import SafeReadEditor from '../../../slate/safe-read-editor';
+import styled from 'styled-components';
+import { VariablePropType } from '../../../../types';
+
+const TableElement = styled.table`
+  width: 100%;
+  border-collapse: collapse;
+  margin: 20px 0;
+`;
+
+const Tr = styled.tr`
+  &:nth-child(even) {
+    background-color: #f2f2f2;
+  }
+`;
+
+const Th = styled.th`
+  background-color: rgb(244, 245, 247);
+  border: 1px solid rgb(193, 199, 208);
+  padding: 8px;
+  min-width: 48px;
+  text-align: center;
```

Comment:
```suggestion
  text-align: left;
```
Agree with @wrynearson that these should be left-aligned